auditlog/admin.py

```diff
@@ -57,3 +57,7 @@ def has_delete_permission(self, request, obj=None):
     def get_queryset(self, request):
         self.request = request
         return super().get_queryset(request=request)
+
+
+if not settings.AUDITLOG_DISABLE_ADMIN_INTERFACE:
```

Comment:
please change the settings name to `AUDITLOG_ENABLE_ADMIN_INTERFACE` with default value of `True`

Reply:
Sure! Makes sense